sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -686,7 +690,7 @@ inline static pi_result createEventAndAssociateQueue(
   // Append this Event to the CommandList, if any
   if (CommandList != Queue->CommandListMap.end()) {
     CommandList->second.append(*Event);
-    PI_CALL(piEventRetain(*Event));
+    (*Event)->RefCount.increment();
```

Comment:
Just curious: why is this change needed?

Reply:
piEventRetain increments external ref count, but we need to increment only internal ref count here. Same logic as for pi_queue.